src/components/shapediver/parameter/ParameterFileInputComponent.tsx

```diff
@@ -20,11 +21,24 @@ export default function ParameterFileInputComponent(props: PropsParameter) {
 		handleChange,
 		onCancel,
 		disabled
-	} = useParameterComponentCommons<File>(props, 0, () => "" /* TODO implement case if default value of File parameter is a string */);
+	} = useParameterComponentCommons<File>(props, 0);
 
 	// create the file endings from all the formats that are specified in the parameter
 	const fileEndings = [...mapMimeTypeToFileEndings(extendMimeTypes(definition.format!))];
 
+	// create a pseudo file in case the value is a file id and a filename for it exists
+	const [ defaultFile, setDefaultFile ] = React.useState<File | null>(null);
+	useEffect(() => {
+		if (typeof(value) === "string" && value.length > 0 && isFileParameter(definition)) {
+			definition.getFilename(value)
+				.then(filename => setDefaultFile(new File([], filename ?? "(Filename unknown)")))
```

Comment:
@mathieu-shapediver do you agree with displaying this string in case the filename is not known? 
<img width="412" alt="image" src="https://github.com/shapediver/ShapeDiverReactExample/assets/8183687/67433e0d-bfa4-4db8-b888-563584099041">

Reply:
@snabela sure, sounds good